test/msw-api/setup-worker/start/on-unhandled-request/suggestions.test.ts

```diff
@@ -0,0 +1,290 @@
+import * as path from 'path'
+import {
+  captureConsole,
+  filterLibraryLogs,
+} from '../../../../support/captureConsole'
+import { runBrowserWith } from '../../../../support/runBrowserWith'
+
+function createRuntime() {
+  return runBrowserWith(path.resolve(__dirname, 'suggestions.mocks.ts'))
+}
+
+describe('REST API', () => {
+  test('does not suggest any handlers when there are no similar ones', async () => {
+    const runtime = await createRuntime()
+    const { messages } = captureConsole(runtime.page)
+
+    await runtime.request({
+      url: runtime.makeUrl('/user-details'),
+    })
+
+    const libraryWarnings = messages.warning.filter(filterLibraryLogs)
+    const unhandledRequestWarning = libraryWarnings.find((text) => {
+      return /\[MSW\] Warning: captured a request without a matching request handler/.test(
+        text,
+      )
+    })
+
+    expect(unhandledRequestWarning).toMatch(`\
+[MSW] Warning: captured a request without a matching request handler:
+
+  • GET /user-details
+
+If you still wish to intercept this unhandled request, please create a request handler for it. Read more: https://mswjs.io/docs/getting-started/mocks.`)
```

Comment:
Nit, should the read more paragraph has its own line?
I'm think that it's easier to catch as a user.

```suggestion
If you still wish to intercept this unhandled request, please create a request handler for it. 
Read more: https://mswjs.io/docs/getting-started/mocks.`)
```